src/mantine-core/src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from 'clsx';
+import React from 'react';
+import useStyles from './Skeleton.styles';
+import { DefaultProps } from '../../theme';
+
+export type SkeletonStylesNames = keyof ReturnType<typeof useStyles>;
+
+export interface SkeletonProps
+  extends DefaultProps<SkeletonStylesNames>,
+    React.ComponentPropsWithRef<'div'> {
+  children?: React.ReactNode;
```

Comment:
Add description to each prop, it will be available at component props tab automatically (you've correctly added description to height prop, do the same thing with all other props)

Reply:
I think the descriptions are there isn't it ?